src/modules/voxelDensity/voxelDensity.h

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// Copyright (c) 2024 Team Dissolve and contributors
+
+#pragma once
+
+#include "base/enumOptions.h"
+#include "classes/atom.h"
+#include "classes/box.h"
+#include "data/atomicMasses.h"
+#include "data/isotopes.h"
+#include "io/export/data1D.h"
+#include "module/module.h"
+#include "templates/array3D.h"
+#include <cmath>
+
+// Forward declarations
+
+class Configuration;
+
+// VoxelDensity Module
+class VoxelDensityModule : public Module
+{
+    public:
+    VoxelDensityModule();
+    ~VoxelDensityModule() override = default;
+
+    /*
+     * Definition
+     */
+    public:
+    // Target property
+    enum class TargetPropertyType
+    {
+        Mass,
+        AtomicNumber,
+        ScatteringLengthDensity
+    };
+    // Return enum option info for TargetPropertyType
+    static EnumOptions<VoxelDensityModule::TargetPropertyType> targetPropertyTypes();
+    // Target property for analysis
+    VoxelDensityModule::TargetPropertyType getCurrentProperty() { return targetProperty_; }
+    // Voxel volume (cubic angstroms)
+    const double voxelVolume() { return voxelVolume_; }
```

Comment:
```suggestion
    double voxelVolume() const { return voxelVolume_; }
```

Did you mean for this to be a const function?  I'm not sure why it would otherwise need to return a `const double`